code/edgeidentity/build.gradle

```diff
@@ -23,14 +25,17 @@ spotless {
 }
 
 android {
-    compileSdkVersion rootProject.ext.compileSdkVersion
+    namespace 'com.adobe.marketing.mobile.edge.identity'
+
+    compileSdk rootProject.ext.compileSdkVersion
 
     defaultConfig {
-        minSdkVersion rootProject.ext.minSdkVersion
-        targetSdkVersion rootProject.ext.targetSdkVersion
+        minSdk rootProject.ext.minSdkVersion
+        targetSdk rootProject.ext.targetSdkVersion
 
         testInstrumentationRunner rootProject.ext.testInstrumentationRunner
         consumerProguardFiles "consumer-rules.pro"
+        multiDexEnabled true
```

Comment:
What does this do, do we need to add this to other extensions?

Reply:
By default, Android apps/modules with minSDKVersion 20 or below are limited to a single dex file.  Currently a single DEX file can only contain 64K method references. This module exceeds that limit. Since the current configuration is not able to fit the (65716) references generated into a single DEX file, the functional tests task fails. `multiDexEnabled` is used to overcome this limitation.

Apps with minSDK >= 21 use ART which allows _Ahead of Time Compilation_ (among other things) and creates an `oat` file by scanning all the DEX files making them multidex enabled by default.